pylint/testutils/configuration_test.py

```diff
@@ -74,18 +70,56 @@ def get_expected_output(
     configuration_path: str, user_specific_path: Path
 ) -> Tuple[int, str]:
     """Get the expected output of a functional test."""
-    output = get_expected_or_default(configuration_path, suffix="out", default="")
-    if output:
+
+    def get_related_files(
+        tested_configuration_file: str, suffix_filter: str
+    ) -> List[Path]:
+        path = Path(tested_configuration_file)
+        return [
+            p
+            for p in path.parent.iterdir()
+            if path.stem in str(p) and str(p).endswith(suffix_filter)
+        ]
+
+    exit_code = 0
+    msg = (
+        "we expect a single file of the form "
+        "'filename_dot_expected_error_code_dot_out.32.out'"
```

Comment:
I think this name is a bit difficult to interpret with the `dot`. I read this as expecting a file with the form of `filename.bad-configuration-section.out.32.out`.

Reply:
c556a5b